tests/topotests/bgp_vpnv4_ebgp/r3/bgpd.conf

```diff
@@ -0,0 +1,25 @@
+router bgp 65501
+ bgp router-id 3.3.3.3
```

Comment:
Could you again replace public routable IPs with private or documentation ranges?

Reply:
done.